srcpkgs/librsvg/template

```diff
@@ -22,13 +24,18 @@ if [ -z "$CROSS_BUILD" ]; then
 	build_options_default="vala gir"
 fi
 
-pre_configure() {
-	autoreconf -fi
+post_configure() {
+	cat <<- EOF > .cargo/config
+[target.${RUST_TARGET}]
+rustflags= ["--sysroot=${XBPS_CROSS_BASE}/usr"]
+EOF
```

Comment:
We have to do this because of https://github.com/rust-lang/cargo/issues/4423

RUSTFLAGS aren't used when `--target` is specified (but they are if we specify target the way we do in the cargo buildstyle: https://github.com/void-linux/void-packages/pull/2837/files#diff-b028c195acdef6080bb12122a8bb6f8dR11